conda_smithy/schema.py

```diff
@@ -1392,7 +1263,7 @@ class ConfigModel(BaseModel):
 
     model = ConfigModel()
 
-    with CONDA_FORGE_YAML_SCHEMA_FILE.open(mode="w+", encoding="utf-8") as f:
+    with CONDA_FORGE_YAML_SCHEMA_FILE.open(mode="w+") as f:
```

Comment:
Why remove this? We just added them consistently all over the place in https://github.com/conda-forge/conda-smithy/pull/2244

Reply:
That was an accident. Will add back!

Reply:
Fixed!